src/partials/partial.erb.html

```diff
@@ -1,4 +1,4 @@
-<div data-key="lmn-book" data-name="<%= customisation.name %>"
+<div data-icons="true" data-key="lmn-book" data-name="<%= customisation.name %>"
```

Comment:
Elaborate? We've managed without modifying the data attributes for everything else.